### DISCUSSION TITLE
Which plugin/config is defining the code spacing?

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#1710] [General]
Example: i want to format my java code with 4 lines but when doing the shortcut for formatting space + c + f, it formats  with 2 lines.

Where do I configure this?

**Comment by dadav** [comment]
I don't use java, but probably some formatter of conform.nvim does that. Find out which formatter it is and adjust the arguments. There is an example for shfmt in the docs: https://www.lazyvim.org/plugins/formatting

**Reply by lkhphuc** [reply]
The new command `:LazyFormatInfo` is exactly what you are asking for.
